starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,44 @@
-// Names and Input
+var hacker1 = "Julien";
+
+console.log('The driver\'s name is ' + hacker1);
+
+var hacker2 = 'Andrea'; 
+
+console.log('The driver\'s name is' + hacker2);
+
+var length1 = hacker1.length;
```

Comment:
it's ok, but you don't need to create a variable for that. It will not modify its value if you modify the ```hacker1``` variable.